components/google/google-maps.md

```diff
@@ -1,3 +1,284 @@
+
+When the map is ready.
+
+##### On Location Changed
+
+![](/assets/google/google-maps/on-location-changed.png)
+
+When the location changes.
+
+##### On Map Click
+
+![](/assets/google/google-maps/on-map-click.png)
+
+When the user clicks on the map.
+
+##### On Map Long Click
+
+![](/assets/google/google-maps/on-map-long-click.png)
+
+When the user long-clicks on the map.
+
+##### On Marker Click
+
+![](/assets/google/google-maps/on-marker-click.png)
+
+When the user clicks on the marker.
+
+##### On Marker Drag
+
+![](/assets/google/google-maps/on-marker-drag.png)
+
+When the user drags a marker.
+
+##### On Marker Drag End
+
+![](/assets/google/google-maps/on-marker-drag-end.png)
+
+When the user ends dragging a marker.
+
+##### On Marker Drag Start
+
+![](/assets/google/google-maps/on-marker-drag-start.png)
+
+When the user starts dragging a marker.
+
+
+
+## Methods
+
+##### Enable Compass
+
+![](/assets/google/google-maps/enable-compass.png)
+
+Enables the compass.
+
+##### Enable Map Camera Pos Change Listener
+
+![](/assets/google/google-maps/enable-map-camera-pos-change-listener.png)
+
+Enables the Map Camera Position Change Listener.
+
+##### Get Markers
+
+![](/assets/google/google-maps/get-markers.png)
+
+Gets markers.
+
+##### Move Camera
+
+![](/assets/google/google-maps/move-camera.png)
+
+Moves the camera.
+
+##### Remove Circle
+
+![](/assets/google/google-maps/remove-circle.png)
+
+Removes the circle.
+
+##### Remove Marker
+
+![](/assets/google/google-maps/remove-marker.png)
+
+Removes the marker.
+
+##### Set Map Type
+
+![](/assets/google/google-maps/set-map-type.png)
+
+Sets the map type.
+
+##### Update Circle
+
+![](/assets/google/google-maps/update-circle.png)
+
+Updates the circle.
+
+##### Update Marker
+
+![](/assets/google/google-maps/update-marker.png)
+
+Updates the marker.
+
+##### Add Overlay
+
+![](/assets/google/google-maps/add-overlay.png)
+
+Adds an overlay.
+
+##### Add Polygon
+
+![](/assets/google/google-maps/add-polygon.png)
+
+Adds a polygon.
+
+##### Add Tile Overlay
+
+![](/assets/google/google-maps/add-tile-overlay.png)
+
+Adds a tile overlay.
+
+##### Clear All Polygons
+
+![](/assets/google/google-maps/clear-all-polygons.png)
+
+Clears all polygons.
+
+##### Draw Central Square
+
+![](/assets/google/google-maps/draw-central-square.png)
+
+Draws a central square.
+
+##### Get My Location
+
+![](/assets/google/google-maps/get-my-location.png)
+
+Gets the users' location.
+
+##### Get Bounding Box
+
+![](/assets/google/google-maps/get-bounding-box.png)
+
+Gets the bounding box.
+
+##### Add Circle
+
+![](/assets/google/google-maps/add-circle.png)
+
+Adds a circle to the map.
+
+##### Add Markers
+
+![](/assets/google/google-maps/add-markers.png)
+
+Adds markers to the map.
+
+##### Add Markers from Json
+
+![](/assets/google/google-maps/add-markers-from-json.png)
+
+Adds markers from json.
+
+##### Add Markers Hue
+
+![](/assets/google/google-maps/add-markers-from-hue.png)

```

Comment:
You uploaded the file as `add-markers-hue.png` not `add-markers-from-hue.png`

Reply:
Made the fix 👍